packages/sui-svg/bin/sui-svg-build.js

```diff
@@ -46,15 +34,13 @@ const getLibFile = file => {
 
 const getAllSrcSvgFiles = () => fg([`${SVG_FOLDER}/**/*.svg`])
 
-const transformSvgToReactComponent = svg =>
-  svgr(svg, {
-    template,
-    expandProps: false,
-    removeTitle: true
-  })
+const transformSvgToReactComponent = svg => {
+  const {data} = optimize(svg)
```

Comment:
This line only optimize the SVG.
After that, we execute template and create a React Component file which uses `dangerouslySetInnerHTML`. 
Given this generated component is a React Component but SVG has original attributes, like `xmlns:xlink` build could fail.

![image](https://user-images.githubusercontent.com/5390428/118268730-0f651200-b4be-11eb-9091-33e6f1d5c821.png)

Reply:
Awesome! Thanks for trying it on MA icons. :) I'll check it!

Reply:
I think this problem is already fixed in latest alpha versions. :) Tried reinsalling it!